app/models/world_location_news.rb

```diff
@@ -52,15 +52,16 @@ def mission_statement
     @content_item.content_item_data.dig("details", "mission_statement")
   end
 
-  def translations
-    @translations ||= @content_item.content_item_data.dig("links", "available_translations")&.map do |translation|
+  def ordered_translations
+    @ordered_translations ||= @content_item.content_item_data.dig("links", "available_translations")&.map do |translation|
       {
         locale: translation["locale"],
         base_path: translation["base_path"],
         text: I18n.t("shared.language_name", locale: translation["locale"]),
         active: I18n.locale.to_s == translation["locale"],
       }
     end
+    @ordered_translations.sort_by { |t| t[:locale] == I18n.default_locale.to_s ? "" : t[:locale] }
```

Comment:
I think there might be an issue here where the cached version of ordered translations is not sorted, so only when we rely on the non-cached version is it sorted. So, we need to cache the sorted stuff as well. 

Suggestion (I think rubocop didn't like my original):

```    
@ordered_translations ||= @content_item.content_item_data.dig("links", "available_translations")&.map { |translation|
      {
        locale: translation["locale"],
        base_path: translation["base_path"],
        text: I18n.t("shared.language_name", locale: translation["locale"]),
        active: I18n.locale.to_s == translation["locale"],
      }
    }&.sort_by { |t| t[:locale] == I18n.default_locale.to_s ? "" : t[:locale] }
```

Reply:
Ended up going with the suggestion above. I didn't realise the caching issue at all.